src/Twig/Extension/ArrayExtension.php

```diff
@@ -188,4 +203,28 @@ public function unique(array $arr1, array $arr2)
 
         return $compiled;
     }
+
+    /**
+     * Tests if a given value is an array or not, returns the result.
+     *
+     * @param mixed $value
+     *
+     * @return bool
+     */
+    public function array($value)
```

Comment:
These can't be used as function names in PHP 5. Plus as discussed in Slack :thinking: over what we do here